docs/how-to/build-policies/rebac/overview.mdx

```diff
@@ -3,124 +3,106 @@ sidebar_position: 1
 
-Role Derivation is the process or mechanism by which new roles are formulated using existing roles as a foundation.
-This can be useful in hierarchical systems where higher-level roles encompass the permissions of several lower-level roles.
-Role derivation helps in creating roles that are a combination or subset of other roles, providing flexibility and ensuring
-that the principle of least privilege is maintained.
+### Resource Roles
 
-#### Example
+To create ReBAC policies, we need to create roles **specific to a given resource**. This means that the role, and the permissions it carries, are only relevant in the context of that specific resource.
+A resource role is formatted as `Resource#Role`.
 
-In a software development company, there might be a basic "Developer" role that grants access to coding tools and
-repositories. A new "Lead Developer" role could be derived from the "Developer" role but with added permissions, like code
-review or project management tools. The "Lead Developer" role would, therefore, have all the permissions of the "Developer"
-role plus some additional ones.
+The combination of **Resource Roles** and **Role Derivations** allows us to derive much more complex and granular roles that are perfectly tailored to handle hierarchies.
 
-### Resource Relation
+:::tipExample
+In the context of our previous example,
 
-Resource Relations refer to the defined relationships or associations between different resources within a system. These
-relations are often used to model complex access control scenarios, especially in systems where access to one resource might
-be dependent on the relationship to another resource. By establishing these relationships, ReBAC can grant or deny access based
-on the context of both the user's attributes and the interconnected web of resources.
+```
+A `user` who is assigned the role of an `Owner` on a `folder`
+```
 
-#### Example
+Will look like this: `Folder#Owner`.
+![Resource Roles](/ui-videos/rebac/2.png)
+:::
 
-In a healthcare system, a patient's medical record `Resource A` might have a `treated by` relationship with a specific doctor
-`Resource B`. Only the doctors that have this "treated by" relation with a patient's record would have the permission to
-access and modify it.
+Relationships are at the core of ReBAC. Here are some of the most common relationship types and how they can be leveraged by ReBAC -
 
-### Resource Instance
+## Common Relationship Types
 
-Resource Instances refer to specific occurrences or representations of a resource in a system. While resource roles or types
-provide a generalized or abstracted idea of an entity, resource instances refer to a specific type of an entity.
-Understanding resource instances is crucial for systems that deal with a multitude of unique entities, each of which might
-have different access requirements.
+### Parent-Child Hierarchies
```

Comment:
Please add couple of words regarding the infinity levels of parent-child hierarchies.